cmake/cmaize/cmaize.cmake

```
@@ -11,4 +11,8 @@ if("${CMAKE_BINARY_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
     message(FATAL_ERROR "${_msg}")
 endif()
 
+include(cmaize/project_specification/project_specification)
+include(cmaize/targets/targets)
 include(cmaize/toolchain/toolchain)
+include(cmaize/project_specification/project_specification)
```

The include shows up twice.
```suggestion
include(cmaize/project_specification/project_specification)
include(cmaize/targets/targets)
include(cmaize/toolchain/toolchain)
```

---

Right, I forgot to fix that after fixing the merge conflict. Thank you!